content/03.results.md

```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
 
 #### Diffuse astrocytic and oligodendroglial tumors
-In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 26) biopsied at diagnosis. 
-The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
-Across HGGs, we found _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) to be the most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
+In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (HGGs, N = 26) biopsied at diagnosis. 
+The single oligodendroglioma sample (N = 1) in the OpenPBTA does not contain mutations in the genes shown and is therefore shown in this oncoprint. 
+Across diffuse astrocytic and oligodendroglial tumors, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
 We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w]. 
-Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**).
-The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
+Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**). **TODO: if we want to reference a figure 5 panel _here_, we need to moe the panel. Otherwise figures are out of order.**
+The two tumors with ultra-high TMB (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
 
 #### Other CNS tumors
-**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors (N = 195). 
-We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
+**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors in the OpenPBTA (N = 195). 
+We observed that 25% (15/60) of ependymoma tumors were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive ependymomas and that 70% (21/30) of craniopharyngiomas were driven by mutations in _CTNNB1_. 
```

Comment:
grant writing tricks - removing extra words ("that") 😄  - we also will need to fall within a certain word count, so may need to remove some of this later, but I haven't even checked this yet!

Reply:
Counterpoint: "that...were" twice is the same number of words as "to be" twice ;)

Reply:
ha!!